### PR TITLE
languages/en-GB - Add missing keys

### PR DESCRIPTION
**Pull request by CloudTheWolf**

### CODE REVIEW

File: languages/en-GB.lua [hunk 1]
```diff
@@ -2470,6 +2476,15 @@ OP.Global.Locales.Languages["en-GB"] = {
...
 
+		givecash_command = "/give_cash",
+		givecash_command_parameter_server_id = "server id",
+		givecash_command_parameter_amoun = "amount",
+		givecash_command_help = "Give another player a certain amount of cash",
+		givecash_command_parameter_server_id_help = "The server id of the player you with to give cash to.",
+		givecash_command_parameter_amount_help = "The amount of cash you want to give to the player",
+		givecash_command_substitutes = "/givecash",
+
+
```
Comment (InZidiuZ) [inline languages/en-GB.lua:2487]: Two new-lines?! Are you insane????

**Review by InZidiuZ — CHANGES_REQUESTED** [review]
I'll update the main language file so you can add all the new ones there instead of just trying to find them from 'missing locales' in-game

**Comment by CloudTheWolf** [comment]
> I'll update the main language file so you can add all the new ones there instead of just trying to find them from 'missing locales' in-game

Thanks I'll get these added now

File: languages/en-GB.lua (hunk 1 above)
Comment (coalaura) [reply languages/en-GB.lua:2487]: This is mayhem

Comment (CloudTheWolf) [reply languages/en-GB.lua]: Fixed!

**Review by InZidiuZ — APPROVED** [review]

**Pull request merged**